src/components/common/Tabs/RectangleProgressTabs/styled.tsx

```diff
@@ -1,15 +1,15 @@
 import { styled } from '@mui/material';
 import { DefaultThemeType } from '../../../../styles/theme/types';
 
-export type TabColorType = 'secondary' | 'error' | 'hidden';
+export type TabColorType = 'default' | 'active' | 'right' | 'error' | string;
```

Comment:
do you need | string here?

Reply:
yes, because will error:  Type 'string' is not assignable to type '"right" | "default" | "active" | "error" | undefined'.